packages/builder/src/stores/builder/builder.ts

```diff
@@ -125,27 +144,27 @@ export class BuilderStore extends BudiStore {
     }
   }
 
-  startBuilderOnboarding() {
+  startBuilderOnboarding(): void {
     this.update(state => ({
       ...state,
       onboarding: true,
       tourKey: TOUR_KEYS.TOUR_BUILDER_ONBOARDING,
     }))
   }
 
-  endBuilderOnboarding() {
+  endBuilderOnboarding(): void {
```

Comment:
Do we always need to specify these `void` return types? I've always left them out because it can trivially work out that it's a void return.

Reply:
Sorry I merged before I noticed your review, ill get them removed in my next PR here